mathesar/api/exceptions/data_import_exceptions/exceptions.py

```diff
@@ -16,3 +16,36 @@ def __init__(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR
     ):
         super().__init__(exception, self.error_code, message, field, details, status_code)
+
+
+class URLDownloadErrorAPIException(MathesarAPIException):
+
+    def __init__(
+            self, exception, error_code=ErrorCodes.URLDownloadError.value,
```

Comment:
@silentninja please use one argument per line consistently, these are three arguments.

Reply:
also applies to other instances